input/pagecontent/index.md

```diff
@@ -1,14 +1,17 @@
 ### Introduction
-This implementation guide is provided to support the use of HL7<sup>&reg;</sup> FHIR<sup>&reg;&copy;</sup> in an Australian context.
+This implementation guide is provided to support the use of HL7<sup>&reg;</sup> FHIR<sup>&reg;&copy;</sup> in an Australian context.  
 
-The AU Base implementation guide presents Australian use concepts defined via FHIR processable artefacts; these are collaborative outputs with agreed approaches to varied kinds of healthcare related information based on the core FHIR R4 specification as base profiles that can be further constrained in a separate implementation guide for a particular usage.
+The AU Base implementation guide defines Australian realm concepts (e.g. Medicare card number) as an additional set of options to what is available in the core FHIR standard, including extensions, terminology and identifiers.  It provides base profiles to inform a reader of which added concepts are considered relevant to a particular resource type, and these base profiles may be further constrained in a separate implementation guide for a particular usage.  For this reason, the AU Base implementation guide does not apply cardinality constraints or required binding strengths to added concepts (except in rare circumstances), utilise must support flags or recommend or mandate any particular resource, element or interactions.
```

Comment:
Remove "the" in front of AU Base, and remove "implementation guide" so it just says "AU Base" instead of 'the AU Base implementation guide' - this needs to be applied across multiple instances.

I think need to reword last sentence to be "... circumstances), and does not utilise must support flags or recommend or mandate any particular resource, element or interactions"

Reply:
Both fixes applied, thanks.